### PR TITLE
update product.html

### PR DESCRIPTION
**Pull request by NataliHavrylenko**

### CODE REVIEW

File: src/partials/product.html [hunk 1]
```diff
@@ -0,0 +1,44 @@
+<section class="products">
+  <div class="container">
+    <p class="products__lable">100% natural</p>
```
Comment (Marrkelov) [inline src/partials/product.html:3]: products__overtitle

File: src/partials/product.html [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    <h2 class="products__main-title">products</h2>
```
Comment (Marrkelov) [inline src/partials/product.html:4]: products__title

File: src/partials/product.html [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    <ul class="products__list">
```
Comment (Marrkelov) [inline src/partials/product.html:5]: product-cards

File: src/partials/product.html [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      <li class="products__item products__item-first list">
```
Comment (Marrkelov) [inline src/partials/product.html:6]: product-cards__item

File: src/partials/product.html [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+        <h3 class="products__item-name">ice cream</h3>
```
Comment (Marrkelov) [inline src/partials/product.html:7]: product-cards__title

File: src/partials/product.html [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+        <p class="products__item-descr">
```
Comment (Marrkelov) [inline src/partials/product.html:8]: product-cards__text

File: src/partials/product.html [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+          Soft ice cream with a delicate taste. Made with milk, cream, sugar and
+          natural stabilizer
+        </p>
+        <button class="arrow-btn" type="button">
```
Comment (Marrkelov) [inline src/partials/product.html:12]: product-cards__button

File: src/partials/product.html [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+          <svg width="40" height="15">
```
Comment (Marrkelov) [inline src/partials/product.html:13]: product-cards__icon

**Comment by Marrkelov** [comment]
Правки

File: src/partials/product.html [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+            <use href="/src/images/sprite.svg#icon-arrow"></use>
```
[the remark this answers is not in the document]
Comment (Marrkelov) [reply src/partials/product.html:14]: подивись в спрайт

[the remark this answers is not in the document]
Comment (NataliHavrylenko) [reply src/partials/product.html:14]: у sprite id для стрілки icon-arrow

File: src/partials/product.html [hunk 10]
```diff
@@ -0,0 +1,44 @@
+<section class="products">
+  <div class="container">
+    <p class="products__overtitle">100% natural</p>
+    <h2 class="products__title">products</h2>
+    <ul class="products__cards">
```
Comment (Marrkelov) [inline src/partials/product.html:5]: product-cards

File: src/partials/product.html [hunk 11]
```diff
@@ -0,0 +1,44 @@
...
+      <li class="product-cards__item product-cards__item-first list">
+        <h3 class="product-cards__title">ice cream</h3>
+        <p class="products__item-text">
```
Comment (Marrkelov) [inline src/partials/product.html:8]: product-cards__text

**Pull request merged**